src/malcolm/malcolmSocketHandler.js

```diff
@@ -29,7 +29,7 @@ const configureMalcolmSocketHandlers = (inputSocketContainer, store) => {
 
   socketContainer.socket.onopen = () => {
     console.log('connected to socket');
-    let path = `${store.getState().malcolm.navigation}`;
+    let path = store.getState().router.location.pathname;
     path = `/gui/${path.split(',').join('/')}`;
```

Comment:
I'm not sure you need the split and join here either, also I'm not sure if the pathname already has `/gui` on it